src/parser/rogue/outlaw/modules/spells/BetweenTheEyesDamageTracker.js

```diff
@@ -9,13 +9,7 @@ class BetweenTheEyesDamageTracker extends FilteredDamageTracker {
   };
 
   shouldProcessEvent(event) {
-    // isOnCooldown returns true when this event is a BTE cast, but we want to keep those casts too
-    if (event.ability.guid !== SPELLS.BETWEEN_THE_EYES.id && this.spellUsable.isOnCooldown(SPELLS.BETWEEN_THE_EYES.id)) {
-      return false;
-    }
-
-    const hasRuthlessPrecision = this.selectedCombatant.hasBuff(SPELLS.RUTHLESS_PRECISION.id);
-    return hasRuthlessPrecision;
+    return !this.spellUsable.isOnCooldown(SPELLS.BETWEEN_THE_EYES.id);
   }
 }
```

Comment:
This doesn't seem right - what are you trying to achieve here?

Reply:
This is to track how many times dispatch was used when BTE was not on CD, in other words, whenever BTE should've been used. This is my first PR, so I'm still learning how the code base works, if this should be done in another way I'd be happy to hear it.

Reply:
The module is a Between The Eyes Damage tracker - surely it should not be tracking Dispatch casts?

Reply:
This should only be tracking Between the Eyes damage. If we want to track improper Dispatch casts we can do that in a separate module.